lib/vanagon/platform/deb.rb

```diff
@@ -29,10 +29,15 @@ def generate_package(project) # rubocop:disable Metrics/AbcSize
       # @param workdir [String] working directory to stage the evaluated templates in
       # @param name [String] name of the project
       # @param binding [Binding] binding to use in evaluating the packaging templates
-      def generate_packaging_artifacts(workdir, name, binding)
+      # @param project [String] Vanagon::Project we are building for
```

Comment:
I'm unclear on what the difference between `name` and `project` is.

Reply:
Ok yeah so this is my fault. name is the name of the project and is a string so for example "puppet-agent" while project is the entirety of a Vanagon::Project and contains a bunch of stuff which we set and is of type Vanagon::Project. I was thinking we passed it in as a String which is definitely not right.

Reply:
ahh, gotcha. Yeah, `Vanagon::Project` seems like the right type. 😊